server/src/db/sql/friend.query.ts

```diff
@@ -0,0 +1,30 @@
+export const GET_FRIENDS = `
+    SELECT u.user_id, u.name, u.email
+    FROM friends f
+    JOIN users u ON f.to_user_id = u.user_id
+    WHERE f.from_user_id = ? AND f.are_we_friend = 1
+    UNION
+    SELECT u.user_id, u.name, u.email
+    FROM friends f
+    JOIN users u ON f.from_user_id = u.user_id
+    WHERE f.to_user_id = ? AND f.are_we_friend = 1`;
```

Comment:
@jjhoooon 
두 쿼리가 거의 동일하므로, OR 조건을 사용하는 하나의 쿼리로 합칠 수 있을 것 같습니다.
```suggestion
export const GET_FRIENDS = `
    SELECT u.user_id, u.name, u.email
    FROM friends f
    JOIN users u ON (f.to_user_id = u.user_id AND f.from_user_id = ?) 
    OR (f.from_user_id = u.user_id AND f.to_user_id = ?)
    WHERE f.are_we_friend = 1`;